styles/style.css

```diff
@@ -0,0 +1,13 @@
+head{
```

Comment:
There won't be anything to style inside the `<head>` elements, so this is not needed in your CSS file. You should remove lines 1-5.